src/plugins/blocksmith/components/instagram-media-block-modal/index.jsx

```diff
@@ -131,12 +133,14 @@ export default class InstagramMediaBlockModal extends React.Component {
       errorMsg,
       hasUpdatedDate,
       hideCaption,
+      id,
       isValid,
       touched,
       updatedDate,
       url,
     } = this.state;
     const { isFreshBlock, isOpen, toggle } = this.props;
+    const displayUrl = isValid ? `https://www.instagram.com/p/${id}/` : url;
```

Comment:
@jalopez13 doesn't `url` already have what you need?

Reply:
@efjacobson 

Updated to use url by itself.